python/tvm/relay/_parser.py

```diff
@@ -252,14 +252,14 @@ def _check_existing_typ_expr(self, name, new_expr):
             new_typ_name = self._type_expr_name(new_expr)
             existing_typ_name = self._type_expr_name(self.global_type_vars[name])
             raise ParseError(
-                f"{new_typ_name} `{name}` conflicts with existing {existing_typ_name}")
+                f"{0} `{1}` conflicts with existing {2}".format({new_typ_name, name, existing_typ_name}))
```

Comment:
From CI Error: 

python/tvm/relay/_parser.py:255: [C0301(line-too-long), ] Line too long (105/100)

@KimBioInfoStudio perhaps you need to make pylint happy?

Reply:
fine, have fix